textproc/libxmlxx5/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           meson 1.0
+
+
+# requires C++11 as of version 2.39.1
+compiler.cxx_standard 2017
+
+name                libxmlxx5
+set gname           libxml++
+version             5.0.1
+set branch          [join [lrange [split ${version} .] 0 1] .]
+categories          textproc
+license             LGPL
+maintainers         openmaintainer
+description         libxml++ is a C++ interface for working with XML files
+long_description    libxml++ is a C++ interface for working with XML files, using \
+                    libxml (gnome-xml) to parse and write the actual XML files. It \
+                    has a simple but complete API.
+homepage            https://libxmlplusplus.github.io/libxmlplusplus/
+platforms           darwin
+distname            ${gname}-${version}
+master_sites        gnome:sources/${gname}/${branch}/
+
+patchfiles-append   patch-meson-build.diff
```

Comment:
if you're patching `meson` to use a Python 3 versions, shouldn't you add a dependency on one?

Reply:
Doesn't meson already depend on python to run? I've patched it such that it always uses whichever Python meson is running on.

Reply:
true, sorry I missed that one - should be all good